website/content/prometheus-statistics/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Prometheus statistics
+weight: 7
+---
+
+## MetalLB K8S client metrics
+
+| Name                                   | Description                                                                      |
+| -------------------------------------- | -------------------------------------------------------------------------------- |
+| metallb_k8s_client_updates_total       | Number of k8s object updates that have been processed                            |
+| metallb_k8s_client_update_errors_total | Number of k8s object updates that failed for some reason                         |
+| metallb_k8s_client_config_loaded_bool  | 1 if the MetalLB configuration was successfully loaded at least once             |
+| metallb_k8s_client_config_stale_bool   | 1 if running on a stale configuration, because the latest config failed to load  |
+
+## MetalLB BGP metrics
```

Comment:
does it makes sense to document the labels as well? i.e in this table you have a "peer" label for each metric (or at least mention this is a per "peer" thing in the title)

Reply:
good idea :)